tests/unit/ssg-module/test_oval_object_model/test_load_and_store.py

```diff
@@ -0,0 +1,323 @@
+import os
+import pytest
+import tempfile
+from ssg.xml import ElementTree
+
+from ssg.oval_object_model import load_oval_document, OVALDocument, OVALEntityProperty
+from ssg.oval_object_model.oval_entities import Criterion, ExtendDefinition
+
+from ssg.xml import open_xml, get_namespaces_from, register_namespaces
+
+
+DATADIR = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "data"))
+OVAL_DOCUMENT_PATH = os.path.join(DATADIR, "oval_with_correct_extend_definition.xml")
+
+PROJECT_ROOT = os.path.join(
+    os.path.dirname(__file__),
+    "..",
+    "..",
+    "..",
+    "..",
+)
+OVAL_DOCUMENT_PATH_FROM_BUILD = os.path.join(
+    PROJECT_ROOT, "build", "ssg-rhel9-oval.xml"
+)
+BUILD_DIR = tempfile.mkdtemp()
+
+
+def _load_oval_document(path):
+    root_el = open_xml(path)
+    ns = get_namespaces_from(path)
+    register_namespaces(ns)
+    return load_oval_document(root_el)
+
+
+def _list_of_shorthands():
+    out = []
+    dir_with_shorthands = os.path.join(DATADIR, "group_dir/rule_dir/oval")
+
+    for file_name in os.listdir(dir_with_shorthands):
+        path_ = os.path.join(dir_with_shorthands, file_name)
+        if not os.path.isdir(path_):
+            with open(path_, "r") as fd:
+                out.append(fd.read())
+
+    return out
+
+
+@pytest.fixture
+def oval_document():
+    return _load_oval_document(OVAL_DOCUMENT_PATH)
+
+
+@pytest.fixture
+def oval_document_from_build():
+    return _load_oval_document(OVAL_DOCUMENT_PATH_FROM_BUILD)
```

Comment:
No, we shouldn't do this, we need to invent a different way. I think that a unit test should never depend on built artifacts. I'm afraid that as the contents of the real products will change the test can start failing.

Reply:
I can see that you already have other tests testing on test data so I think that you should remove this and move the checks to the test cases that test on the the test data.